pkg/ranchhand/tools.go

```diff
@@ -19,18 +19,26 @@ type RequiredToolURLs struct {
 }
 
 // todo: need a better way structure for managing tool versions
-var PlatformToolURLs = map[string]RequiredToolURLs{
-	"darwin": {
-		Kubectl: "https://storage.googleapis.com/kubernetes-release/release/v1.13.3/bin/darwin/amd64/kubectl",
-		Helm:    "https://storage.googleapis.com/kubernetes-helm/helm-v2.12.3-darwin-amd64.tar.gz",
-		RKE:     "https://github.com/rancher/rke/releases/download/v0.1.17/rke_darwin-amd64",
-	},
-	"linux": {
-		Kubectl: "https://storage.googleapis.com/kubernetes-release/release/v1.13.3/bin/linux/amd64/kubectl",
-		Helm:    "https://storage.googleapis.com/kubernetes-helm/helm-v2.12.3-linux-amd64.tar.gz",
-		RKE:     "https://github.com/rancher/rke/releases/download/v0.1.17/rke_linux-amd64",
-	},
-}
+var (
+	PlatformToolVersions = map[string]string{
+		"kubectl": "v1.14.0",
+		"helm":    "v2.13.1",
+		"rke":     "v0.2.0",
```

Comment:
Does rke 0.2.0 install k8s 1.14 by default? If not, should we try to make our kubectl match the k8s version? Also makes me wonder if we should be targeting a specific k8s release explicitly in the rke config.

Reply:
- The default k8s version for [rke v0.2.0](https://github.com/rancher/rke/releases/tag/v0.2.0) is the same as v0.17.0 (previous version).
- We could explicitly target a release but I found that every subsequent rke release substantially changes its supported major/minor versions that it would require code edits in multiple places. Maybe this desirable?
- Thankfully, `kubectl` is largely backwards compatible; one minor version is a non-issue. I just thought it would be useful to have all the newest bugfixes and whiz-bang available in the newest version of this CLI tool.

Reply:
re:tracking the rke versions, I think you're probably right and it isn't worth bothering with at this point in time.

re:the other stuff, I just thought the 1.14 kubectl was curious and wanted to raise the obvious questions, but I don't have any major worries. LGTM!